wicket-core/src/main/java/org/apache/wicket/markup/head/CssReferenceHeaderItem.java

```diff
@@ -154,7 +154,13 @@ private String getUrl()
 	@Override
 	public int hashCode()
 	{
-		return Objects.hash(super.hashCode(), reference, getMedia(), pageParameters, getRel());
+		String s;
+		int result = super.hashCode();
+		result = 31*result + ((reference != null) ? reference.hashCode() : 0);
```

Comment:
There should be whitespace around the multiplier. Please apply our standard code format.